schemas/main.go

```diff
@@ -61,10 +61,6 @@ func realMain() error {
 		return fmt.Errorf("version is required for GHES platform")
 	}
 
-	if (platform == platformType.String(dotcom) || platform == platformType.String(ghec)) && version != "" {
```

Comment:
This error was removed to simplify logic in the `scripts/generate-{whatever}.sh` script: there, we can specify the version every time through an environment variable, and then simply ignore it here if it's blank.